ui/src/app/layout/topics/single-topic.component.ts

```diff
@@ -1,17 +1,23 @@
-import { Component, OnInit, ViewEncapsulation } from '@angular/core';
-import { routerTransition } from '../../router.animations';
-import { ActivatedRoute, Router } from '@angular/router';
-import { SchemaMetadata, Topic, TopicRecord, TopicsService, TopicSubscription } from '../../shared/services/topics.service';
-import { combineLatest, Observable } from 'rxjs';
-import { finalize, flatMap, map, shareReplay, startWith, take } from 'rxjs/operators';
-import { ApplicationsService, UserApplicationInfo } from '../../shared/services/applications.service';
-import { EnvironmentsService, KafkaEnvironment } from '../../shared/services/environments.service';
-import { ToastService } from '../../shared/modules/toast/toast.service';
-import { TranslateService } from '@ngx-translate/core';
-import { NgbDateStruct, NgbModal } from '@ng-bootstrap/ng-bootstrap';
-import { CertificateService } from '../../shared/services/certificates.service';
+import {Component, OnInit, ViewEncapsulation} from '@angular/core';
+import {routerTransition} from '../../router.animations';
+import {ActivatedRoute, Router} from '@angular/router';
+import {
+    SchemaMetadata,
+    Topic,
+    TopicRecord,
+    TopicsService,
+    TopicSubscription
+} from '../../shared/services/topics.service';
+import {combineLatest, Observable} from 'rxjs';
+import {finalize, flatMap, map, shareReplay, startWith, take} from 'rxjs/operators';
+import {ApplicationsService, UserApplicationInfo} from '../../shared/services/applications.service';
+import {EnvironmentsService, KafkaEnvironment} from '../../shared/services/environments.service';
+import {ToastService} from '../../shared/modules/toast/toast.service';
+import {TranslateService} from '@ngx-translate/core';
+import {NgbDateStruct, NgbModal} from '@ng-bootstrap/ng-bootstrap';
+import {CertificateService} from '../../shared/services/certificates.service';
 import * as moment from 'moment';
```

Comment:
Please adjust your formatter to not reformat these imports.

Reply:
done.